spring-boot/admin/src/main/resources/templates/admin/fragments/scripts.html

```diff
@@ -1,4 +1,4 @@
-<html xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity4"
```

Comment:
Did you remove the namespace on purpose? Is it not required?

Reply:
No, I didn't remove it. I will add this again on the next commit